src/composables/use-usage-session-id.js

```diff
@@ -0,0 +1,22 @@
+import { ssrRef } from '@nuxtjs/composition-api'
+import { v4 as uuidv4 } from 'uuid'
+
+/**
+ * Sets the usageSessionId as a per-request ssrRef. The value is created on the server
+ * in one of the layout components, and is saved to the store to be used when outside
+ * the component context.
+ * TODO: refactor to use Pinia store during the media store refactor.
+ *
+ * @param store
+ * @returns {{ usageSessionId: string }}
+ */
+export function useUsageSessionId(store) {
+  let usageSessionId = ssrRef('')
```

Comment:
I think this also needs a key for us to be able to use multiple `ssrRef`s, doesn't it? Something like 

```suggestion
  const usageSessionId = ssrRef('', 'usage-session-id')
```

The documentation for this function is so unclear it's really frustrating.

Also we can update it to a const instead of let, right?

Reply:
You need a key for multiple `ssrRef`'s , but I don't think we are going to use any more `ssrRef`'s anywhere? And when we go to Nuxt bridge/3, we will replace `ssrRef` with `useState` (keyed).

I agree about documentation, I added the key at first, but could not find how to use the keyed ssrRef, so I removed the key in the end. And yes, it should be `const`.